prepare/tasks/llm_as_judge/model_response_assessment.py

```diff
@@ -0,0 +1,12 @@
+from unitxt.blocks import FormTask
+from unitxt.catalog import add_to_catalog
+
+add_to_catalog(
+    FormTask(
+        inputs=["question", "model_output"],
+        outputs=["rating_label"],
+        metrics=["metrics.spearman"],
+    ),
+    "tasks.llm_as_judge.rag.model_response_assessment",
```

Comment:
why is it rag? this is more general

Reply:
@OfirArviv - it assumes the input is called "question" which is relevant to rag and qa.